project/BuildDependencies/DownloadMingwBuildEnv.bat

```diff
@@ -1,63 +1,568 @@
-@ECHO OFF
+::-------------------------------------------------------------------------------------
+:: LICENSE -------------------------------------------------------------------------
+::-------------------------------------------------------------------------------------
+::  This Windows Batchscript is for setup a compiler environment for building ffmpeg and other media tools under Windows.
+::
+::    Copyright (C) 2013  jb_alvarado
+::
+::    This program is free software: you can redistribute it and/or modify
+::    it under the terms of the GNU General Public License as published by
+::    the Free Software Foundation, either version 3 of the License, or
+::    (at your option) any later version.
+::
+::    This program is distributed in the hope that it will be useful,
+::    but WITHOUT ANY WARRANTY; without even the implied warranty of
+::    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+::    GNU General Public License for more details.
+::
+::    You should have received a copy of the GNU General Public License
+::    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+::-------------------------------------------------------------------------------------
 
-SETLOCAL
+@echo off
+color 80
```

Comment:
can we get rid of this? Screws up my console colors.
Right is how it usually looks, left is after running this.
![image](https://cloud.githubusercontent.com/assets/874927/13204072/b13e751e-d8c6-11e5-9bc0-d3c94a74e6fa.png)